pkgs/applications/system/mgmt/default.nix

```diff
@@ -0,0 +1,71 @@
+{ augeas
+, buildGoModule
+, fetchFromGitHub
+, gotools
+, lib
+, libvirt
+, libxml2
+, nex
+, pkg-config
+, ragel
+}:
+buildGoModule rec {
+  pname = "mgmt";
+  version = "unstable-2022-10-24";
+
+  src = fetchFromGitHub {
+    owner = "purpleidea";
+    repo = pname;
+    rev = "d8820fa1855668d9e0f7a7829d9dd0d122b2c5a9";
+    hash = "sha256-jurZvEtiaTjWeDkmCJDIFlTzR5EVglfoDxkFgOilo8s=";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile --replace "/usr/bin/env " ""
```

Comment:
I am pretty sure that the sandbox contains those directories.

The following more likely means that the command after env is not available.

```
make: /usr/bin/env: No such file or directory
make: /usr/bin/env: No such file or directory
make: /usr/bin/env: No such file or directory
make: /usr/bin/env: No such file or directory
make: /usr/bin/env: No such file or directory
make: /usr/bin/env: No such file or directory
make: /usr/bin/env: No such file or directory
make: /usr/bin/env: No such file or directory
make: /usr/bin/env: No such file or directory
make: /usr/bin/env: No such file or directory
```

Reply:
I have added `bash` as a `nativeBuildInput`, and rebuilt everything with `nixpkgs-review pr 198963` and it finished cleanly.

if this is still not working, i do not know how to debug it further. can you point me to the logs or how to reproduce it locally?

Reply:
Click on details on the ofborg checks next to the github actions or build the package locally

Reply:
I have built the package locally and ofborg seems to be saying everything is kosher